lib/models/node.js

```diff
@@ -64,6 +69,31 @@ function NodeModelFactory (Model, waterline, _) {
                     return !_.isEmpty(catalog);
                 });
             }
+        },
+        addTags: function(id, tags) {
+            return Promise.fromNode(this.native.bind(this))
```

Comment:
finally spotted the `native` here - guessing it'll be easier to catch you on chat to ask about this, but any pointers you can help me with are appreciated. is the `Promise.fromNode` what promisifyies the `this.native` as you're binding it?